app/models/service/aggregation.rb

```diff
@@ -42,26 +151,36 @@ def aggregate_direct_vm_memory_on_disk
   end
 
   def aggregate_all_vm_cpus
-    all_vms.inject(0) { |aggregate, vm| aggregate += vm.cpu_total_cores.to_i }
+    attributes["aggregate_all_vm_cpus"] || all_vms.inject(0) { |aggregate, vm| aggregate += vm.cpu_total_cores.to_i }
   end
 
   def aggregate_all_vm_memory
-    all_vms.inject(0) { |aggregate, vm| aggregate += vm.ram_size.to_i }
+    if attributes["aggregate_all_vm_memory"]
+      # Avoids (poorly) "ERROR:  integer out of range" in postgres
+      attributes["aggregate_all_vm_memory"] * 1.megabyte
+    else
+      all_vms.inject(0) { |aggregate, vm| aggregate += vm.ram_size.to_i }
+    end
   end
 
   def aggregate_all_vm_disk_count
-    all_vms.inject(0) { |aggregate, vm| aggregate += vm.num_disks.to_i }
+    attributes["aggregate_all_vm_disk_count"] || all_vms.inject(0) { |aggregate, vm| aggregate += vm.num_disks.to_i }
```

Comment:
Discussed I think in https://github.com/ManageIQ/manageiq/pull/11502#discussion_r80530332, but I can switch to that if you would prefer.